src/Maker/AdminMaker.php

```diff
@@ -205,7 +205,7 @@ private function generateService(
             $controllerName = $this->controllerClassBasename ? $controllerClassFullName : '~';
 
             $id = $input->getOption('id') ?:
-                $this->getAdminServiceId('App', $this->adminClassBasename);
```

Comment:
Isn't this a BC-break ?

Reply:
No, `getAdminServiceId` method is private